openarena-q3e

```diff
@@ -1,29 +1,59 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+fi
+# Use the value of RESPECT_XDGDIRS to determine the directory
+if [ "$RESPECT_XDGDIRS" -eq 1 ]; then
+    # Use ~/.var/app/io.github.ec_.Quake3e.OpenArena/.openarena/ directory
+    mkdir -p "$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/homepath"
```

Comment:
Why not using the "$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/data" directory instead? It is persistent by default and should not have any problems. And since it ia a standard directory, you can just use `${XDG_DATA_HOME}` variable instead of this hacky solution.

Reply:
Also the `${XDG_DATA_HOME}` directory is created by default by the Flatpak and does not need to be created manually.